app/controllers/api/students/comments_controller.rb

```diff
@@ -2,6 +2,7 @@ class Api::Students::CommentsController < Api::BaseController
 
   def create
     comment = Comment.new comment_params
+    comment.student_id = params[:student_id]
```

Comment:
Can you do me a favor and just try `comment.student = params[:student_id]`? I have a feeling the convention is that this magic should work.